src/encoders/path_encoder.py

```
@@ -0,0 +1,128 @@
+    print(f"Path: {example_path}")
+    print(f"Encoding: {encoding}")
+    print(f"  Depth: {int(encoding[0])} (3 levels deep)")
+    print(f"  Hash bucket: {int(encoding[1])}")
+    
+    # More examples
+    test_paths = [
+        "/main.py",                    # Root level
+        "/src/app.py",                 # 1 level deep
+        "/src/utils/helpers.py",       # 2 levels deep  
+        "/src/core/models/user.py",    # 3 levels deep
+        "/docs/api/endpoints/users.md" # 3 levels deep
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)